manifests/galactic.capital.investment.yaml

```
@@ -0,0 +1,13 @@
+name: galactic.capital.investment 
+authors: zuckung
+homepage: https://github.com/zuckung/endless-sky-plugins/
+license: MIT
+version: v1.1
```

```suggestion
version: v1.1-galactic.capital.investment
```
analogous to your tag name - it's a bit ugly, but works